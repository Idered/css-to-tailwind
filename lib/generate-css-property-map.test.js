const generateCssPropertyMap = require('./generate-css-property-map');

test('generateCssPropertyMap works', async () => {
    expect(await generateCssPropertyMap()).toMatchSnapshot();
})