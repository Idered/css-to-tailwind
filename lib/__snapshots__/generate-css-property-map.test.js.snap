// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateCssPropertyMap works 1`] = `"foo"`;
